draft-ietf-quic-recovery.md

```diff
@@ -837,37 +837,43 @@ approximately equivalent to two TLPs before an RTO in TCP.
 This duration is computed as follows:
 
 ~~~
-(smoothed_rtt + max(4 * rttvar, kGranularity) + max_ack_delay) *
-    kPersistentCongestionThreshold
+PTO * kPersistentCongestionThreshold
 ~~~
 
-For example, assume:
+where PTO is computed as described in {{computing-pto}}.
+
+The following example illustrates how persistent congestion can be
+established. Assume:
 
 ~~~
-smoothed_rtt = 1
-rttvar = 0
-max_ack_delay = 0
+PTO = 1.5
 kPersistentCongestionThreshold = 3
 ~~~
 
-If an ack-eliciting packet is sent at time t = 0, the following scenario would
-illustrate persistent congestion:
-
-| Time | Action                 |
-|:-----|:-----------------------|
-| t=0  | Send Pkt #1 (App Data) |
-| t=1  | Send Pkt #2 (PTO 1)    |
-| t=3  | Send Pkt #3 (PTO 2)    |
-| t=7  | Send Pkt #4 (PTO 3)    |
-| t=8  | Recv ACK of Pkt #4     |
-
-The first three packets are determined to be lost when the acknowledgement of
-packet 4 is received at t = 8.  The congestion period is calculated as the time
-between the oldest and newest lost packets: (3 - 0) = 3.  The duration for
-persistent congestion is equal to: (1 * kPersistentCongestionThreshold) = 3.
-Because the threshold was reached and because none of the packets between the
-oldest and the newest packets are acknowledged, the network is considered to
-have experienced persistent congestion.
+Consider the following sequence of events:
+
+| Time  |          Action            |
+|:------|:---------------------------|
+| t=0   | Send packet #1 (app data)  |
+| t=1   | Send packet #2 (app data)  |
+| t=1.2 | Recv acknowledgement of #1 |
+| t=2   | Send packet #3 (app data)  |
+| t=3   | Send packet #4 (app data)  |
+| t=4   | Send packet #5 (app data)  |
+| t=5   | Send packet #6 (app data)  |
+| t=6.5 | Send packet #7 (PTO 1)     |
+| t=9.5 | Send packet #8 (PTO 2)     |
+| t=9.7 | Recv acknowledgement of #8 |
+
+Packets 2 through 7 are declared lost when the acknowledgement for packet 8 is
+received at t = 9.7.
+
+The congestion period is calculated as the time between the oldest and newest
+lost packets: 6.5 - 1 = 5.5.  The duration for establishing persistent
+congestion is: 1.5 * 3 = 4.5.  Because the threshold was reached and because
+none of the packets between the oldest and the newest lost packets were
+acknowledged, the network is considered to have experienced persistent
+congestion.
```

Comment:
I think we should still add a line here about PTO not being necessary for establishing persistent congestion since the example still uses PTO.

Reply:
There is a change in Jana's other PR.